release_notes/ocp-4-12-release-notes.adoc

```diff
@@ -722,81 +748,166 @@ In the table below, features are marked with the following statuses:
 |TP
 |TP
 
+|====
+
+[id="ocp-installation-technology-preview-features"]
+=== Installing technology preview features
```

Comment:
This sort of reads like a sentence (like, how to install TP features). Maybe "Installation Technology Preview features" instead?

Reply:
+1